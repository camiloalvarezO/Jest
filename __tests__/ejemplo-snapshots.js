const persona ={
    nombre: "cami",
    balance: 500,
    tipo :'premiun'
}

describe("testing al cliente",()=>{
    test("haciendole un snapshot al cliente",()=>{
        expect(persona).toMatchSnapshot(); // para crear otro snapshot se puede hacer npm t -- -u
        // jest deber√≠a ser el unico que toque los snapshots, mejor eliminar la carpeta/archivo 
        // y hacer otro
        
    })
})