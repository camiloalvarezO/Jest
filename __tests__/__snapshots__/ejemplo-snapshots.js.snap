// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing al cliente haciendole un snapshot al cliente 1`] = `
{
  "balance": 500,
  "nombre": "cami",
  "tipo": "premiun",
}
`;
